packages/blaze/view.js

```diff
@@ -184,6 +184,34 @@ Blaze.View.prototype.autorun = function (f, _inViewScope) {
   return c;
 };
 
+/**
+ * Just like Blaze.View#autorun, but with Meteor.subscribe instead of
+ * Tracker.autorun. Stop the subscription when the view is destroyed.
+ * @return {SubscriptionHandle} A handle to the subscription so that you can
+ * see if it is ready, or stop it manually
+ */
+Blaze.View.prototype.subscribe = function (/* arguments */) {
+  var self = this;
+
+  if (! self.isCreated) {
+    throw new Error("View#subscribe must be called from the created callback at the earliest");
+  }
+  if (this._isInRender) {
```

Comment:
If I understand correctly, these two if clauses (and the one above I guess) are really an attempt to say: "if not in a created or rendered callback throw an error". Is there a better way to check for this? (I don't know the answer). It looks  a little ad-hoc.

At the very least I'd recommend extracting these into a utility function called, say, "canCallSubscribe" that's documented as checking if you're in a created or rendered callback with a comment saying this is the best way you found to check for this.

Maybe @dgreensp has a better suggestion.